src/pages/DrumMachinePage.spec.js

```diff
@@ -0,0 +1,11 @@
+import DrumMachine from './DrumMachinePage';
+import { render } from '@testing-library/react';
+
+
+
+describe('DrumMachine', () => {
+  it('renders a section with sixteen buttons', () => {
+    render(<DrumMachine />);
+
```

Comment:
I'm not sure, if you need a test here

Reply:
yes, it doesn't make much sense like that, but I'll test all the buttons here now, as already written above